### DISCUSSION TITLE
Welcome to fully-homomorphic-encryption Discussions!

### DISCUSSION THREAD

**Discussion by j2kun** [google/fully-homomorphic-encryption#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FlorentCLMichel** [comment]
Hi 👋
That's a great initiative! Thanks @j2kun for creating this discussion space!
For my part, I'm currently working on hardware acceleration for FHE. I have used the transpiler for doing experiments using different algorithms and I find it a really fantastic tool to bridge the knowledge gap between software engineering and FHE - which is certainly one of the major issues (along with speed) to be solved for FHE to become widely used. 
I am mostly involved in low-level lattice operations used in FHE and their hardware acceleration, but also interested in how they integrate in higher-level workflows and how we can, as a community, make it easier for people outside the Cryptography space to use FHE.
